app/views/companies/show.html.haml

```diff
@@ -29,15 +29,18 @@
           %h2 #{t('.address')}:
           = field_or_none t('.street'), @company.main_address.street_address, tag_options: {class: 'street'}
 
+          = field_or_none t('.post_code'), @company.main_address.post_code, tag_options: {class: 'post-code'}
+
           = field_or_none t('.city'), @company.main_address.city, tag_options: {class: 'city'}
 
           = field_or_none t('.kommun'), @company.main_address.kommun.name, tag_options: {class: 'kommun'}
```

Comment:
Good info.  (Those makandra cards and a good resource.)  I'll also check to see if the `safe operator` should be used in other ones.